pkg/tfbridge/info.go

```diff
@@ -99,39 +99,47 @@ type ProviderInfo struct {
 	PreConfigureCallbackWithLogger PreConfigureCallbackWithLogger
 }
 
+// Describe the mapping from resource and datasource tokens to Pulumi resources and
+// datasources.
+//
+// NOTE: Experimental; We are still iterating on the design of this function, and it is
+// subject to change without warning.
+type ComputeDefaultInfo struct {
```

Comment:
Nit: I don't like the *Info name. It puts it as a synonym of ResourceInfo in my head which is not as helpful. Perhaps. DefaultsStrategy?

Reply:
Changed in https://github.com/pulumi/pulumi-terraform-bridge/pull/766/commits/8145571811581e9e848bcaccbba2527b956079a0